.github/workflows/rust_binaries_release.yml

```diff
@@ -4,12 +4,20 @@ on:
   push:
     branches:
       - master
-      - 'releases/**'
-      - 'hotfix/**'
```

Comment:
can we restore this check for `pull_request`?

Reply:
yeah, that's what I meant.

```yml
    if: startsWith(github.head_ref, 'releases/')
```

or something like that, just to maintain the behaviour